index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Main Css file -->
+    <link rel="stylesheet" href="css/master.css" />
+    <!-- Render all elements normally -->
+    <link rel="stylesheet" href="css/Normalize.css" />
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio Website: Mohammed Raad</title>
+  </head>
+  <body>
+    <!-- Toolbar start -->
+    <nav id="nav">
+      <div class="nav-container container">
+        <div class="name">Mohammed Raad</div>
+        <div class="links">
+          <span class="icon">
+            <span></span>
+            <span></span>
+            <span></span>
+          </span>
+        </div>
+      </div>
+    </nav>
+    <!-- Toolbar end -->
+    <!--  headline section start -->
+    <section id="headline">
+      <div class="headline-container container">
+        <div class="text">
+          <h1>Hey There. I’m Mohammed</h1>
+          <h3>I’am a Software Developer</h3>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="social-icon">
+            <img src="imgs/github.png" alt="github" />
+            <img src="imgs/linkedin.png" alt="linkedin" />
+            <img src="imgs/hand.png" alt="none" />
+            <img src="imgs/twitter.png" alt="twitter" />
+            <img src="imgs/M.png" alt="M" />
+          </div>
+        </div>
+      </div>
+      <div class="image">
+        <img src="imgs/Illustration.png" alt="background" />
```

Comment:
- [ ] Nice work done on the headline section, but kindly move the background image to the extreme right and make sure it doesn't cover the text so it matches the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)